libraries/core_libs/network/rpc/jsonrpc_http_processor.hpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
```

Comment:
why dont we have headers from rpc folder in include/rpc and sources in src/rpc ?